decisions/mnm_hermetics_decisions.txt

```diff
@@ -338,7 +338,8 @@ targeted_decisions = {
 		}
 	}
 
-	#Power: write Magnum Opus
+	# Warcraft
+	#Power: write down whispers
 	hermetics_write_magnum_opus = {
 		is_in_society = yes
```

Comment:
```suggestion
		# Warcraft
		only_rulers = yes
		
		is_in_society = yes
```
...since the books written by courtiers will end in liege's treasury.